validator/src/contains.rs

```diff
@@ -40,8 +40,12 @@ impl<'a, S> Contains for &'a HashMap<String, S> {
 /// Validates whether the value contains the needle
 /// The value needs to implement the Contains trait, which is implement on String, str and Hashmap<String>
 /// by default.
-pub fn validate_contains<T: Contains>(val: T, needle: &str) -> bool {
-    val.has_element(needle)
+pub fn validate_contains<T: Contains>(val: T, needle: &str) -> Result<(), String> {
+    if val.has_element(needle) {
+        Ok(())
+    } else {
+        Err(format!("must contain '{}'", needle))
```

Comment:
I don't want the messages to be in the `validator` crate, they should be in the `validator_derive`.
Validator fns should just return a bool